src/js/components/Menu.js

```diff
@@ -149,6 +160,13 @@ var MenuDrop = React.createClass({
     if (this.props.small) {
       classes.push(CLASS_ROOT + "__drop--small");
     }
+    if (this.props.className) {
+      var classNames = this.props.className.split(" ");
+      classNames = classNames.map(function(className) {
+        return className + "__drop";
```

Comment:
It doesn't seem right to append __drop to all classes provided. This would include whatever classes the caller provides to Menu.